examples/.env

```diff
@@ -0,0 +1,3 @@
+    ROOT_PASSWD=<root_password>
```

Comment:
Can you put this line in quotes so it's technically valid like the other ones?

Reply:
Sure